setup.cfg

```diff
@@ -0,0 +1,19 @@
+[metadata]
+name = psql2mysql
+version = 0.4.0
+summary = Copy data from PostgreSQL databases to MySQL
+home-page = https://github.com/SUSE/psql2mysql
+license = Apache-2
+classifier =
+    Development Status :: 3 - Alpha
+    Intended Audience :: Developers
+    License :: OSI Approved :: Apache License 2.0
+    Programming Language :: Python :: 2
+    Programming Language :: Python :: 2.7
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.5
```

Comment:
should we include 3.5 in the travis matrix?

Reply:
we should